recipes/openalea-deploy/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "openalea.deploy" %}
+{% set version = "2.1.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/openalea/deploy/archive/v{{ version }}.tar.gz
+  sha256: a2163cd662f5f2eb2062abc58982201a0241c94d84d28bfdca5388a78ac6ef22
+
+build:
+  number: 0
+  script: "{{ PYTHON }} setup.py install --single-version-externally-managed --record record.txt"
+  preserve_egg_dir: True
+
+requirements:
+  host:
+    - python
+    - setuptools
+  run:
+    - python
+    - setuptools
+    - pywin32  # [win]
+    - six
+    - path.py
+
+
+test:
+  imports:
+    - openalea.deploy
+
+about:
+  home: http://github.com/openalea/deploy
+  license: CECILL-C
+  license_file: LICENSE
+  summary: OpenAlea.Deploy support the installation of OpenAlea packages.
+
+  # The remaining entries in this section are optional, but recommended.
+  description: |
+    OpenAlea.Deploy support the installation of OpenAlea packages for Python 2.7+ and
+    Python 3.3+. It is an extension of setuptools.
+    Features :
+
+        Discover and manage packages in EGG format
+        Install shared libraries, include and libs in eggs
+        Support 'develop' command
+        Call SCons or CMake scripts in setup.py
+        Create namespaces if necessary
+
+  dev_url: https://github.com/openalea/deploy
+
+extra:
+  recipe-maintainers:
+    - pradal
+    - fredboudon
```

Comment:
@fredboudon are you OK with being a maintainer?

Reply:
Yes. @fredboudon has already accepted previously : https://github.com/conda-forge/staged-recipes/pull/10370#issuecomment-563403365